### ISSUE TITLE
If the app is running, building of the release fails

### ISSUE THREAD

**Issue by janjiss** [bitwalker/exrm#142]
Hey there, 
I encountered this bug which seems to happen if we have a running server with active erlang pipes in /tmp directory. 
```
mix stdout: ==> Building release with MIX_ENV=prod.
==> Generating relx configuration...
==> Generating sys.config...
==> Generating boot script...
==> Conform: Loading schema...
==> Conform: No schema found, conform will not be packaged in this release!
==> Performing protocol consolidation...
==> Generating release...
==> Generated .appup for elixir_stream 0.0.1 -> 0.0.2
/usr/local/bin/mix:2: syntax error before: '#'
/usr/local/bin/mix:2: syntax error before: '#'
/usr/local/bin/mix:2: syntax error before: '#'
/usr/local/bin/mix:2: syntax error before: '#'
/usr/local/bin/mix:2: syntax error before: '#'
/usr/local/bin/mix:2: syntax error before: '#'
/usr/local/bin/mix:2: syntax error before: '#'
/usr/local/bin/mix:2: syntax error before: '#'
/usr/local/bin/mix:2: syntax error before: '#'
===> relup successfully created!
==> Generating nodetool...
==> Packaging release...
mix stderr: ** (MatchError) no match of right hand side value: {:error, {:bad_file_type, '/var/www/myapp/releases/20150626075959/rel/myapp/tmp/erl_pipes/myapp/erlang.pipe.1.r', :other}}
    lib/mix/tasks/release.ex:358: Mix.Tasks.Release.update_release_package/1
    lib/mix/tasks/release.ex:73: Mix.Tasks.Release.do_run/1
    (mix) lib/mix/cli.ex:55: Mix.CLI.run_task/2
    (elixir) lib/code.ex:307: Code.require_file/2
```

**Comment by janjiss** [comment]
Okay, after some digging, I've found out that this part of the code fails. If I exclude tmp directory when filtering trough list of compressable files, it works like a charm. 
```
:ok = :erl_tar.create(
      '#{tarball}',
      file_list ++ extras,
      [:compressed]
    )
```
Should I create pull request for that? I am not sure about the consequences this might have.

**Comment by bitwalker** [comment]
@janjiss Ah yes we definitely weren't accounting for that properly, if you can create a PR I'll certainly merge it!

**Comment by bitwalker** [comment]
Closed by #144
